Packages/MIES/MIES_IgorHooks.ipf

```diff
@@ -167,6 +167,8 @@ static Function IH_Cleanup()
 
 		DFREF dfrNWB = GetNWBFolder()
 		KilLVariables/Z dfrNWB:histRefNumber
+
+		ASSERT(!ASYNC_WaitForWLCToFinishAndRemove(WORKLOADCLASS_URL, 60), "JSON Payload upload did not finish within timeout")
```

Comment:
A 60-second timeout may not be enough because upload speeds can be slow. If there isn't any harm in waiting longer, I suggest 5 minutes. @t-b Feel free to merge after considering this.

Reply:
Changed it to 5 minutes. Note: This timeout only kicks in if the uploading is not finished when closing MIES.